MathlibTest/TacticAnalysis.lean

```diff
@@ -59,3 +61,20 @@ example : 1 + 1 = 2 := by
   rfl
 
 end replaceWithGrind
+
+end declarations
+
+section internals
+
+section setOptionIn
+
```

Comment:
Can you add a short comment describing what this is testing? (So future users don't accidentally change the test away from what it meant to test.)

Reply:
Done! :) [36ce532](https://github.com/leanprover-community/mathlib4/pull/28926/commits/36ce532d9eab83bcf7ce4b75c2fecc80d07c72c6)